### PR TITLE
Remove dependency on bower for tests

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: test/spec/expose.js [hunk 1]
```diff
@@ -29,18 +29,12 @@ asyncTest("AMD support", 1, function() {
...
-      underscore: "bower_components/underscore/underscore",
-      jquery: "bower_components/jquery/dist/jquery",
-      backbone: "bower_components/backbone/backbone"
+      underscore: "node_modules/underscore/underscore",
+      jquery: "node_modules/jquery/dist/jquery",
+      backbone: "node_modules/backbone/backbone"
     },
 
     shim: {
-      underscore: { exports: "_" },
```
Comment (SimenB) [inline test/spec/expose.js]: Both Underscore and Backbone support UMD, no reason to shim them

File: test/spec/expose.js [hunk 2]
```diff
@@ -29,18 +29,12 @@ asyncTest("AMD support", 1, function() {
...
-      backbone: {
-        deps: ["jquery", "underscore"],
-        exports: "Backbone"
-      },
-
       "backbone.layoutmanager": useLM || {
```
Comment (SimenB) [inline test/spec/expose.js]: Not sure if this is needed?

Comment (tbranyen) [inline test/spec/expose.js]: Are you asking about line 38? If so, try removing it.  If the tests still pass, we don't need it ;-)

Comment (SimenB) [inline test/spec/expose.js]: Heh, fair point. Removed now :smile:

**Comment by tbranyen** [comment]
Awesome! Nice work :)

**Pull request merged**